src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEPropertySymbol.vb

```diff
@@ -45,6 +46,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         Private _lazyObsoleteAttributeData As ObsoleteAttributeData = ObsoleteAttributeData.Uninitialized
 
         Private _lazyIsRequired As ThreeState = ThreeState.Unknown
+        Private _lazyOverloadResolutionPriority As StrongBox(Of Integer)
```

Comment:
>StrongBox [](http://example.com/codeflow?start=51&length=9)

Should we add an `UncommonFields` class to hold most of the `_lazy*` fields?

Reply:
I don't think we have to, and I decided against doing this  work under umbrella of this feature.